testing/tests/DevExpress.exporter/imageCreator.tests.js

```diff
@@ -2652,3 +2557,66 @@ QUnit.test('Some items is async', function(assert) {
         done();
     });
 });
+
+QUnit.module('Tests with private API usage', {
+    beforeEach() {
+        this.orig_getBlob = imageCreator._getBlob;
+        this.orig_getBase64 = imageCreator._getBase64;
+
+        imageCreator._getBlob = () => 'blobData';
+        imageCreator._getBase64 = () => 'base64Data';
+    },
+    afterEach() {
+        imageCreator._getBlob = this.orig_getBlob;
+        imageCreator._getBase64 = this.orig_getBase64;
+    }
+});
+
+QUnit.test('getData returns Blob when it supported by Browser', function(assert) {
+    if(!typeUtils.isFunction(window.Blob)) {
+        assert.ok(true, 'Skip if there isn\'t blob');
+        return;
+    }
+
+    const done = assert.async();
+    const markup = '<svg></svg>';
+
+    const deferred = imageCreator.getData(markup, { backgroundColor: '#aaa' });
+
+    assert.expect(1);
+    $.when(deferred).done(function(data) {
+        assert.equal(data, 'blobData', '_getBlob was called');
```

Comment:
please, use `strictEqual` everywhere instead of `equal`. You don't use `==`, right?